actix-web-codegen/README.md

```diff
@@ -3,7 +3,7 @@
 > Helper and convenience macros for Actix Web
 
 [![crates.io](https://meritbadge.herokuapp.com/actix-web-codegen)](https://crates.io/crates/actix-web-codegen)
-[![Documentation](https://docs.rs/actix-web-codegen/badge.svg)](https://docs.rs/actix-web)
+[![Documentation](https://docs.rs/actix-web-codegen/badge.svg)](https://docs.rs/actix-web-codegen/0.4.0/actix_web_codegen/)
```

Comment:
The original file was pointing to `actix_web`, if this was intentional I could revert it

Reply:
no that was an oversight, thanks